### DISCUSSION TITLE
Italian Translation

### DISCUSSION THREAD

**Discussion by phox081** [vuejs-translations/guidelines#23] [General]
👋  Ciao a tutti,
c'è qualcuno che vorrebbe iniziare una traduzione italiana per la nuova documentazione di Vue3?

👋  Hi all,
is there anyone who would like to start an Italian translation for the new Vue3 documentation?

2023 update: 

- https://github.com/vuejs-translations/guidelines/discussions/23#discussioncomment-6603795
- https://github.com/vuejs-translations/guidelines/discussions/23#discussioncomment-6833634

**Comment by BlackdestinyXX** [comment]
> 👋 Ciao a tutti, c'è qualcuno che vorrebbe iniziare una traduzione italiana per la nuova documentazione di Vue3?
> 
> 👋 Hi all, is there anyone who would like to start an Italian translation for the new Vue3 documentation?

Si, io sarei interessato

**Comment by tommasinigiovanni** [comment]
Anche io sarei interessato

**Comment by phox081** [comment]
Ciao, 
bene stiamo aumentando 🙂
Potremmo cominciare ad organizzarci.
A presto

**Comment by leonardocarlas** [comment]
Ciao ragazzi, io mi unirei volentieri, come siete messi? Siete partiti alla fine?

**Comment by francescogropplero** [comment]
Buongiorno! Anch'io sarei interessato a partecipare. Avete mai iniziato anche solo a discuterne?

**Reply by BlackdestinyXX** [reply]
Ciao, purtroppo no, almeno non penso. Magari qualcuno di noi potrebbe cominciare a creare una repo così possiamo iniziare.

**Comment by leonardocarlas** [comment]
Ciao a tutti!
Siete riusciti a creare la repo? Altrimenti posso guardare io in questi
giorni.

Leonardo

Il giorno ven 7 apr 2023 alle ore 17:39 Francesco Gropplero Di Troppenburg <
***@***.***> ha scritto:

**Comment by phox081** [comment]
Ciao a tutti, 
@tommasinigiovanni , @leonardocarlas , @BlackdestinyXX  , @Mattia-Zanini 

sto riprendendo il progetto per la traduzione di vuejs docs v3. 
Mi scuso per l'assenza e le mancate risposte. 

Questo periodo di assenza, però, ha un lato positivo: la documentazione ufficiale ora sembra abbastanza stabile e non dovremo fare modifiche sostanziali sui file principali appena tradotti. 

Sto preparando uno schema ed un indice che ci aiuteranno nell'organizzazione del lavoro, con la speranza di velocizzare e semplificare gli sforzi di tutti quelli che vorranno partecipare.

La repo IT per ora è qui: [https://github.com/phox081/docs-it](https://github.com/phox081/docs-it)
Il progetto è hostato in demo su netlify e viene rilasciato ad ogni merge sul `Main` [https://it-vuejs-org.netlify.app/](https://it-vuejs-org.netlify.app/).

Per quelli che vogliono partecipare e per qualsiasi necessità scrivetemi, 
ho anche attivato un canale discord dove possiamo scriverci e sentirci: [https://discord.com/channels/486549696584876033/1135155463411998770](https://discord.com/channels/486549696584876033/1135155463411998770)

**Reply by francescogropplero** [reply]
Ciao @phox081 ! Che piacere vedere la cosa prendere forma. Siccome avevo ritenuto che mi avrebbe fatto piacere fare parte della cosa mi sono portato avanti, scaricando la repo e traducendo già qualche pagina del dropdown About. Ho seguito la guida readme ed ho clonato la repo su una personale e creato un branch separato chiamato TranslateBranch, dove ho fatto il push delle pagine tradotte, puoi trovare tutto sul mio profilo GitHub. 
Purtroppo il link del discord non funziona, se riesci creane un'altro in modo di facilitare la comunicazione fra di noi! Grazie buona serata

**Reply by phox081** [reply]
Ciao @francescogropplero, ottimo, se sei on board ti invito subito alla repo come Contributor, così possiamo lavorare più velocemente senza fare troppi fork.

Per la parte About, la guardo in mattinata e vediamo come procedere. Come per altri progetti converrebbe creare una Issue per ogni pagina/sezione, così da avere un unico punto dove scrivere e collegare le pr e i merge a quella issue, questo faciliterà il lavoro di revisione a tutti quelli che si uniscono in corsa.

Attualmente io mi sto concentrando sulla parte della "Guida" e non dovremmo sovrapporci. 

Come accennato in precedenza ho preparato delle liste di file/sezioni per procedere in modo organizzato, penso che il modo più comodo per condividerlo sia in una discussione nel progetto, vi aggiornerò appena pubblico.

Per il canale Discord a me il link sembra funzionare, qui il link per invito: https://discord.gg/BbxPxg6N (il link scade in 7gg).
Se anche così hai problemi prova a condividere il tuo utente discord e ti aggiungo a mano (su discord sono: paxmk)
 
A più tardi.

**Reply by ilpaolino** [reply]
Ciao @phox081 , sarei interessato anche io a partecipare, è possibile?

**Reply by phox081** [reply]
Ciao @ilpaolino, certo. Oggi sono in giro. Riesci a collegarti al server Discord? Cosi ci organizziamo.

**Reply by albim360** [reply]
Ciao!
Mi servirebbe il nuovo link del server discord, inoltre ho aperto una discussione sulla tua repo in merito ad un mio errore che nonostante tutto spero possa tornare utile ahah.

Mi piacerebbe collaborare.

**Reply by phox081** [reply]
ciao @albim360, il link è questo: https://discord.gg/wERMdxhR contattami pure

**Reply by Efois** [reply]
Vorrei inserirmi. E' possibile?

**Comment by phox081** [comment]
@yyx990803 , @Jinjiang  , @edimitchel , @skirtles-code 

Hello everyone,
We are excited to officially announce that we have started translating the Vue.js documentation into Italian |🇮🇹. We have made significant progress and are proud to inform you that we are close to completing 50% of the content.

A special thanks goes to the contributors for their support. We invite anyone interested to join us or provide feedback. We will continue to work diligently to finish the translation and ensure it is of high quality.
Thank you for your attention and support!

The IT Repo can be found here: [https://github.com/phox081/docs-it](https://github.com/phox081/docs-it)
The Netlify preview is available here: [https://it-vuejs-org.netlify.app](https://it-vuejs-org.netlify.app/)/

Ciao 👋,
[phox081](https://github.com/phox081)

--------------------------

Ciao a tutti,
Siamo felici di annunciare ufficialmente che abbiamo iniziato la traduzione della documentazione di Vue.js in italiano |🇮🇹. Abbiamo fatto notevoli progressi e siamo orgogliosi di comunicarvi che siamo vicini al completamento del 50% del contenuto.
Un ringraziamento speciale va ai contributors per il loro supporto. Invitiamo tutti coloro che sono interessati a unirsi a noi o a fornire feedback. Continueremo a lavorare sodo per completare la traduzione e garantire che sia di alta qualità.
Grazie per la vostra attenzione e il supporto!

Il repo IT è qui: [https://github.com/phox081/docs-it](https://github.com/phox081/docs-it)
L'anteprima Netlify è qui: [https://it-vuejs-org.netlify.app](https://it-vuejs-org.netlify.app/)/

Ciao 👋,
[phox081](https://github.com/phox081)

**Reply by Jinjiang** [reply]
@phox081 great job! Congrats! I already invited you into the org. After accepting that, you would be able to transfer the repo here and maintain it further. Thanks.

**Reply by phox081** [reply]
Hi @Jinjiang ,
Thank you so much! I'm truly excited and have already accepted the invitation. We have a few PRs open in draft, and I'd like to close them before proceeding with the repo transfer. I hope that's okay with you. Thanks for your quick response and support.

Ciao,
@phox081

**Reply by phox081** [reply]
Hi @Jinjiang I have made the transfer to vuejs-translations/docs-it, now we will proceed to finalize the last issues. What are the next steps?

**Reply by Jinjiang** [reply]
@phox081 Cool. Glad to know that. We will create an official deployment on it.vuejs.org later. At the same time, please help to "detach your fork". See more details here: https://github.com/vuejs-translations/guidelines#high-level-steps-of-working-on-a-new-translation
Thanks.
/cc @yyx990803 

Update: https://it.vuejs.org/ is ready.

**Reply by phox081** [reply]
Hi @Jinjiang, 
I tried to perform the detach using the link proposed, but I couldn't complete the operation because the Github Virtual Agent says the repo "doesn't exist or you are not an admin." I have opened a support ticket and I am waiting for a response. Is there something else we can do?

**Reply by phox081** [reply]
Hi @Jinjiang, 

Just a quick update about detaching the repo - I've sent you an email with all the details and steps provided by GitHub Support.

Let me know how I can assist further.

ciao,
Pax - phox081

**Reply by Jinjiang** [reply]
@phox081 Gotcha. Thanks for your information. No worries. I created a ticket for that. Will keep you posted.

**Reply by Jinjiang** [reply]
Update: the repo has been detached.
Now feel free to create a Pull Request to add a link to the translation on vuejs.org

**Reply by phox081** [reply]
Hi @Jinjiang,

done, PR is here: https://github.com/vuejs/docs/pull/2533

We are going to refine and upload the last pages in the coming weeks.

**Comment by albim360** [comment]
Ciao a tutti!
Sarei lieto di contribuire alla traduzione!

**Comment by Frapenle** [comment]
Hi, is it possible to have the discord channel link? I'd like to participate in the translation

**Comment by phox081** [comment]
> Hi, is it possible to have the discord channel link? I'd like to participate in the translation

Ciao @Frapenle prova con questo link:

https://discord.com/channels/486549696584876033/1135155463411998770

**Reply by Frapenle** [reply]
Ciao @phox081, il link non funziona

**Comment by Efois** [comment]
> Ciao a tutti, @tommasinigiovanni , @leonardocarlas , @BlackdestinyXX , @Mattia-Zanini
> 
> sto riprendendo il progetto per la traduzione di vuejs docs v3. Mi scuso per l'assenza e le mancate risposte.
> 
> Questo periodo di assenza, però, ha un lato positivo: la documentazione ufficiale ora sembra abbastanza stabile e non dovremo fare modifiche sostanziali sui file principali appena tradotti.
> 
> Sto preparando uno schema ed un indice che ci aiuteranno nell'organizzazione del lavoro, con la speranza di velocizzare e semplificare gli sforzi di tutti quelli che vorranno partecipare.
> 
> La repo IT per ora è qui: https://github.com/phox081/docs-it Il progetto è hostato in demo su netlify e viene rilasciato ad ogni merge sul `Main` https://it-vuejs-org.netlify.app/.
> 
> Per quelli che vogliono partecipare e per qualsiasi necessità scrivetemi, ho anche attivato un canale discord dove possiamo scriverci e sentirci: https://discord.com/channels/486549696584876033/1135155463411998770

**Comment by Spanu18** [comment]
@phox081 @Jinjiang 
👋 Ciao a tutti, vorrei contribuire alla traduzione della documentazione. Ho creato una pull request, ma ho notato che la repository sembra inattiva o non mantenuta. C'è qualche modo per continuare la traduzione?

👋 Hello everyone, I would like to contribute to the documentation translation. I created a pull request, but I noticed that the repository seems inactive or unmaintained. Is there any way to continue the translation?

**Reply by ilpaolino** [reply]
Ciao, 
in genere scriviamo per coordinarci su discord, (link https://discord.gg/jsGA3Jr6 ) hai seguito l'iter scritto nel file CONTRIBUTING? 
https://github.com/vuejs-translations/docs-it/blob/main/CONTRIBUTING.md 

Se il sistema me lo permette provo a prendere in mano io la PR nei prossimi giorni.

Ciao Paolo
